static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -267,6 +294,9 @@ export function DraggableTabBar({
       });
       setTabs(newTabs);
       tabListState?.setSelectedKey(tempId);
+      trackAnalytics('issue_views.add_view_clicked', {
```

Comment:
i think some of these names are slightly different : `issue_views.add_view_clicked` vs `issue_views.add_view.clicked`

Reply:
ah my bad, I updated that in the definition and forgot to change it in the usages